bin/_config.sh

```diff
@@ -60,25 +58,28 @@ function _docker_run() {
         i=$(( i+1 ))
     done
 
-    # Check that minishift is running
-    if ${OC_LOGIN} && ! minishift status | grep Running > /dev/null ; then
-        echo "Error: minishift is not running. Please start it first."
+    # Check that the current user is already logged in OpenShift
+    if ! oc whoami &> /dev/null; then
```

Comment:
Some scripts using Arnold (such as `pytest`) do not require to be logged in to OpenShift, and, thus are not impacted by the `K8S_AUTH_HOST` or `OPENSHIFT_DOMAIN` values. Hence, I think we must keep the `OC_LOGIN` flag functional:

```bash
if ${OC_LOGIN} && ! oc whoami &> /dev/null; then
    # ...
```